README.md

```diff
@@ -16,8 +16,6 @@ for further details.
 
 For questions about the model, use [Github Discussions](https://github.com/E3SM-Project/E3SM/discussions).
 
-See our Github-hosted documentation at [https://e3sm-project.github.io/E3SM/](https://e3sm-project.github.io/E3SM/).
```

Comment:
Why removing this? I often want to get to our generated docs, and never remember the correct github.io link. Having it on the main page of the project makes it easy to reach it.